db/migrate/20230815163624_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :email
+      t.string :photo
+
+      t.timestamps
+    end
```

Comment:
[REQUIRED] Kindly include bio and posts_counter just as it is in the ERD diagram

Reply:
I have added the bio and the counter. All is up to date now.